frontend/src/app/pages/Profile/index.js

```diff
@@ -27,10 +28,12 @@ class ProfileComponent extends Component {
         <Subtitle>Elo rating: {rating}</Subtitle>
         <ProfileDetail>
           <Subtitle>Matches: {totalMatches}</Subtitle>
+          <Subtitle>Wins: {Math.round(totalMatches * winRatio)}</Subtitle>
```

Comment:
I guess it would have to be like hundred or thousands of matches for the rounding error to possibly appear but I agrre it is a bad practise so I changed it to [here](https://github.com/salsita/foosball-rating/pull/114/commits/3d398530cdf75f980e6344e885e4f18dd919311a#diff-08117b849ecc00af7c3c0d609dd86d31)